src/main/java/tutoraid/model/lesson/Name.java

```diff
@@ -0,0 +1,54 @@
+package tutoraid.model.lesson;
```

Comment:
Wonder if this should be renamed to LessonName to avoid confusion with, say, StudentName

Reply:
Renamed it to LessonName, thanks.